src/main/java/com/zerobase/SelfWash/member/application/SignInApplication.java

```diff
@@ -0,0 +1,56 @@
+package com.zerobase.SelfWash.member.application;
+
+import static com.zerobase.SelfWash.member.domain.type.MemberType.ADMIN;
+
+import com.zerobase.SelfWash.config.security.JwtProvider;
+import com.zerobase.SelfWash.member.components.MailComponent;
+import com.zerobase.SelfWash.member.domain.form.AdminSignUpForm;
+import com.zerobase.SelfWash.member.domain.form.SignInForm;
+import com.zerobase.SelfWash.member.domain.form.SignUpForm;
+import com.zerobase.SelfWash.member.domain.type.MemberType;
+import com.zerobase.SelfWash.member.service.signin.AdminSignInService;
+import com.zerobase.SelfWash.member.service.signin.SignInService;
+import com.zerobase.SelfWash.member.service.signup.AdminSignUpService;
+import com.zerobase.SelfWash.member.service.signup.SignUpService;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class SignInApplication {
+
+  private final List<SignInService> signInServices;
+  private final AdminSignInService adminSignInService;
+
+  public void signIn(MemberType memberType, SignInForm signInForm) {
+    SignInService signInService = getSignInService(memberType);
+    signInService.signIn(signInForm);
```

Comment:
```suggestion
    getSignInService(memberType).signIn(signInForm);
```
불필요한 선언은 없어도 될것 같습니다